frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -83,14 +132,19 @@ async function parsePrismaSchema(schemaString: string): Promise<ProcessResult> {
             deleteConstraint: 'NO_ACTION',
           } as const)
 
-      relationships[relationship.name] = relationship
+      relationships[relationship.name] = getFieldRenamedRelationship(
+        relationship,
+        tableFieldRenaming,
+      )
     }
   }
   for (const index of dmmf.datamodel.indexes) {
     const table = tables[index.model]
     if (!table) continue
 
-    const indexInfo = extractIndex(index)
+    const indexInfo = extractIndex(
+      getFieldRenamedIndex(index, tableFieldRenaming),
+    )
```

Comment:
`indices` has names which consists of field name (i.e. `user_email_key `). It's hard to convert after build a liam-index object because it requires to modify the names, so I add a function to apply renaming to MDDF-index object to rename field name before creating liam-index.